frontend/src/routes/ManageAdmissions/CreateAdmission.tsx

```diff
@@ -131,6 +131,19 @@ const CreateAdmission: React.FC = () => {
             onChange={formik.handleChange}
           />
         </InputWrapper>
+        <InputWrapper>
+          <InputTitle>Slug</InputTitle>
+          <InputDescription>
+            Opptaket vil ligge under opptak.abakus.no/[slug]/
+          </InputDescription>
+          <Input
+            name="slug"
+            value={formik.values.slug}
+            onChange={formik.handleChange}
+            placeholder="f. eks. komiteer/revy/backup"
+            disabled={!isNew}
+          />
+        </InputWrapper>
```

Comment:
I'm assuming validation errors are handled by the backend here?

Reply:
Yas (at least the basics for a slug, haven't added more than that), had some problems with the way the frontend does requests regarding fetching error data, so I'm awaiting refactoring that code to display the errors properly in the frontend.

Although I have completely forgotten to add frontend validation at all to that form - so I'll write that up to take a look at improving it later(: